eldev.el

```diff
@@ -4907,10 +4907,13 @@ possible to build arbitrary targets this way."
                  files-to-tar)
             (condition-case nil
                 (make-symbolic-link eldev-project-dir (expand-file-name name-version working-dir))
-              (file-error (error "FIXME: Copy")))
+               (file-error
+                 (let ((working-dir-pkg (expand-file-name name-version working-dir)))
+                   (copy-directory eldev-project-dir working-dir-pkg t t t))))
             (make-directory (file-name-directory package-target) t)
             (unless (file-exists-p descriptor-file)
-              (with-temp-file descriptor-file
+              (with-temp-file (concat (file-name-as-directory working-dir)
```

Comment:
Use `expand-file-name` instead of `concat`.

Reply:
Done.